Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -1380,3 +1671,5 @@ def main():
 
 if __name__ in ['__main__', 'builtin', 'builtins']:
     main()
+
+# register_module_line('SentinelOne V2', 'end', __line__())
```

Comment:
```suggestion
```
Please remove these lines

Reply:
@MLainer1  - I accidentally left in my comments to remove the `register_module_line` function calls - which relates to your suggested change. In general, my understanding is that they should be there, but that caused pytest to fail. So confirming you want them completely removed?

Also, the remaining validations that are failing are due to YAML issues in playbooks. The YAML was generated by XSOAR. It seems more likely that I would introduce errors if I try to manually resolve each of those. But if you are saying I should try to add fields that it is complaining about, I will take a stab at it. Let me know.

Reply:
@[CraigWampler](https://github.com/CraigWampler) I'll look into the yml validations failure, but they will still have to be resolved to be able to merge your pr, so it maybe necessary to generate the yml file again (or add the fields  manually)